ssl/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "display_name": "SSL",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "ssl",
+  "metric_prefix": "ssl_cert.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Monitor SSL certificates for expiration, validity, etc.",
+  "guid": "4e27a211-a034-42dd-9939-9ef967b1da50",
+  "support": "core",
+  "supported_os": ["linux", "mac_os", "windows"],
+  "public_title": "Datadog-SSL Integration",
+  "categories": [""],
```

Comment:
Let's add this at least to `network` but maybe the same thing as in the http intg